python/lsst/obs/decam/decamFilters.py

```diff
@@ -45,6 +45,8 @@
                      alias={'Y'}),
     FilterDefinition(physical_filter="VR DECam c0007 6300.0 2600.0",
                      band="VR",
+                     doc='A very broad-band filter, intended for "discovery", not "accurate photometry".'
```

Comment:
Now I'm wondering what VR means.  "Very R..."?  And is band of "VR" a reasonable thing -- is VR a generic concept?

Reply:
My guess is that it covers both V and R: it's ~500-750nm. Sadly, the webpage I linked doesn't specify where the name comes from. I don't think it's a generic concept, but we'd want to ask a DECam person what they'd call the "band" for this one, if we don't want to go with that. Certainly, "vr" is unlikely to be a confusing band name elsewhere.